projects/program-website/src/app/projects-component/projects.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
I'm not sure you know it, but the unit test actually does nothing. It contain some many cryptic code and it's useless.
My opinion, we shouldn't keep it, unless you add some real tests here.
Even if the test is created automatically by `ng`.